pkg/spoke/controllers/statuscontroller/availablestatus_controller.go

```diff
@@ -187,58 +201,88 @@ func aggregateManifestConditions(generation int64, manifests []workapiv1.Manifes
 	}
 }
 
+func (c *AvailableStatusController) getFeedbackValues(
+	resourceMeta workapiv1.ManifestResourceMeta, obj *unstructured.Unstructured, manifestOptions []workapiv1.ManifestConfigOption) ([]workapiv1.FeedbackValue, error) {
```

Comment:
obj can be nil when it is not found.   should this method shortcircuit on that?

Reply:
good point, added a check before calling this function